packages/ui/package.json

```diff
@@ -1,24 +1,87 @@
 {
   "name": "@penumbra-zone/ui",
   "version": "3.5.0",
-  "private": true,
+  "private": false,
   "license": "(MIT OR Apache-2.0)",
   "type": "module",
+  "engine": {
+    "node": ">=22"
+  },
   "scripts": {
+    "build": "tsup",
```

Comment:
thought: interestingly, I wouldn't even think about using tsup for a UI build but it seems to be working just fine. Thanks for showing a way to do it!